app/renderer/webviewDisplay.js

```diff
@@ -0,0 +1,51 @@
+function ensurePaintWebviewFirstAttach (webview) {
+  window.requestAnimationFrame(() => {
+    webview.style.display = 'none'
+    window.requestAnimationFrame(() => {
+      webview.style.display = ''
+    })
+  })
+}
+
+function ensurePaintWebviewSubsequentAttach (webview) {
+  window.requestAnimationFrame(() => {
+    webview.style.top = '1px'
+    window.requestAnimationFrame(() => {
+      webview.style.top = ''
+    })
+  })
+}
+
+module.exports = class SingleWebviewDisplay {
+  constructor ({containerElement, classNameWebview}) {
+    this.isAttached = false
+    this.webview = this.createWebview()
+    this.webview.classList.add(classNameWebview)
+    containerElement.appendChild(this.webview)
+  }
+
+  attachActiveTab (tabId) {
+    console.log('webviewDisplay: attaching tab id', tabId)
```

Comment:
extra console.log?

Reply:
Thanks for the spot. I actually haven't done a pass of cleanup or lint fixing as we're still debugging a couple display issues.